packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeModel.ts

```diff
@@ -135,9 +139,10 @@ export namespace TreeModel {
   }
 
   export function setInstanceFilter(model: TreeModel, nodeId: string | undefined, filter?: GenericInstanceFilter): boolean {
```

Comment:
at this point `setInstanceFilter` is nearly identical to `setHierarchyLimit` - could we extract common pieces into a shared function?